### PR TITLE
chore: migrate default branch from master to main

### PR DESCRIPTION
**Pull request by leahecole**

### CODE REVIEW

**Review by dandhlee — CHANGES_REQUESTED** [review]

**Review by dandhlee — COMMENTED** [review]
One more, almost there!

**Comment by parthea** [comment]
Hi @leahecole, Please could you take a look?

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ with your document storage on Google Cloud Storage.
 - `Product Documentation`_
 
 .. |GA| image:: https://img.shields.io/badge/support-GA-gold.svg
-   :target: https://github.com/googleapis/google-cloud-python/blob/master/README.rst#general-availability
+   :target: https://github.com/googleapis/google-cloud-python/blob/main/README.rst#general-availability
```
[the remark this answers is not in the document]
Comment (parthea) [reply README.rst]: Done.

**Comment by parthea** [comment]
@googlebot I consent.

**Review by parthea — APPROVED** [review]

**Comment by parthea** [comment]
Hi @dandhlee, Please can you take another look?

**Pull request closed without merging**